src/irmin/store.ml

```diff
@@ -233,6 +233,7 @@ module Make (B : Backend.S) = struct
     type t = repo
 
     let v = B.Repo.v
+    let config (t : t) = B.Repo.config t
```

Comment:
```suggestion
    let config = B.Repo.config
```
Small nit. There is a mixture of styles in this section of code, but I'm partial to this suggestion. If you keep it as-is, is it possible to drop the type specification?